src/utils.js

```diff
@@ -338,75 +338,77 @@ async function inputActionNamePrompt (actionName, isRetry) {
   return answer.actionName
 }
 
+// ----The following functionality is disabled until the @adobe/aio-cli-plugin-extension is updated. It is currently not behaving as expected.----//
+
 /**
  * Checks if @adobe/aio-cli-plugin-extension is already installed
  */
-async function isPluginExtensionInstalled () {
-  const oclifConfig = await Config.load(path.dirname(path.dirname(fs.realpathSync(process.argv[1]))))
-  const pluginsRegistry = new Plugins.default(oclifConfig)
-  const plugins = await pluginsRegistry.list()
-  const isInstalled = plugins.some(plugin => plugin.name === PLUGIN_EXTENSION)
+// async function isPluginExtensionInstalled () {
+//   const oclifConfig = await Config.load(path.dirname(path.dirname(fs.realpathSync(process.argv[1]))))
+//   const pluginsRegistry = new Plugins.default(oclifConfig)
+//   const plugins = await pluginsRegistry.list()
+//   const isInstalled = plugins.some(plugin => plugin.name === PLUGIN_EXTENSION)
 
-  return isInstalled
-}
+//   return isInstalled
+// }
 
 /**
  * Prompts for installing @adobe/aio-cli-plugin-extension if not already installed
  */
-async function installPluginExtension () {
```

Comment:
The PR removes mention of the plugin and prompts to install it, but the auto-registration error will still occur if the user has the plugin installed before running the template because the template still outputs `app.config.yaml` files with `event-listener-for` properties

I don't know that the `event-listener-for` property is used for anything other than supporting auto-registration creation. Has remove it from files generated using the template been discussed?